build/params_mainnet.go

```diff
@@ -79,7 +79,8 @@ const UpgradeOhSnapHeight = 1594680
 // 2022-07-06T14:00:00Z
 const UpgradeSkyrHeight = 1960320
 
-var UpgradeSharkHeight = abi.ChainEpoch(99999999999999)
+// 2022-11-30T14:00:00Z
+var UpgradeSharkHeight = abi.ChainEpoch(2383680)
```

Comment:
why not constant?

Reply:
god took my 5 min to get the exact epoch lolol

Reply:
Because we have LOTUS_DISABLE_SHARK below that changes the value to max int if set